perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/impl/SchedulingPoolImpl.java

```diff
@@ -146,7 +147,24 @@ public void setPerun(Perun perun) {
 		this.perun = perun;
 	}
 
-
+	// --- session init ----------------------------------
+	
+	public void afterPropertiesSet() {
```

Comment:
There is a missing `@Override` annotation.

Reply:
Added.